### PR TITLE
Added serializer function 

### PR DESCRIPTION
**Pull request by ljeyaratnam**

### CODE REVIEW

File: src/main/java/org/atlasapi/client/GsonQueryClient.java [hunk 1]
```diff
@@ -289,21 +289,28 @@ public T read(JsonReader in) throws IOException {
...
         @Override
         public DateTime deserialize(JsonElement json, Type typeOfT,
                 JsonDeserializationContext context) throws JsonParseException {
             return formatter.parseDateTime(json.getAsString());
         }
+
+        @Override
+        public JsonElement serialize(DateTime src, Type typeOfSrc,
+                JsonSerializationContext context) {
+            return new JsonPrimitive(this.printer.print(src));
```
Comment (MartinsIrbe) [inline src/main/java/org/atlasapi/client/GsonQueryClient.java]: I'm not sure about this, don't we need to convert DateTime object to millis? Also some tests would be helpful ;)

Comment (tfm) [inline src/main/java/org/atlasapi/client/GsonQueryClient.java]: A `DateTime` should be serialized as a ISO-8601 datetime, not millis.

Comment (tfm) [inline src/main/java/org/atlasapi/client/GsonQueryClient.java]: No need to use `this.` here

File: src/main/java/org/atlasapi/client/GsonQueryClient.java [hunk 2]
```diff
@@ -289,21 +289,28 @@ public T read(JsonReader in) throws IOException {
...
                 }
-                
+
             };
         }
     }
 
-    public static final class DateTimeDeserializer implements JsonDeserializer<DateTime> {
-        
+    public static final class DateTimeDeserializer implements JsonDeserializer<DateTime>, JsonSerializer<DateTime> {
```
Comment (Dalamar42) [inline src/main/java/org/atlasapi/client/GsonQueryClient.java]: This is not longer just a Deserializer so it should be renamed appropriately.

File: src/test/java/org/atlasapi/client/GsonQueryClientTest.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+public class GsonQueryClientTest {
+
+    @Mock JsonSerializationContext context;
+
+    @Test
+    public void serializeDateTime() {
+        DateTime dateTime = new DateTime();
+        GsonQueryClient.JodaDateTimeSerializer serializer = new GsonQueryClient.JodaDateTimeSerializer();
+        JsonElement element = serializer.serialize(dateTime, DateTime.class, context);
+        assertEquals(dateTime.toString(), element.getAsString());
```
Comment (tfm) [inline src/test/java/org/atlasapi/client/GsonQueryClientTest.java]: Be more explicit here. It's not obvious to a reader of the test what format you're asserting is expected. `assertEquals("2016-03-09T00:00:00Z")` would make it clear.

**Comment by tfm** [comment]
One minor comment, otherwise LGTM

**Pull request merged**